external/netutils/netlib_getipv4addr.c

```diff
@@ -101,6 +101,7 @@ int netlib_get_ipv4addr(FAR const char *ifname, FAR struct in_addr *addr)
 		if (sockfd >= 0) {
 			struct ifreq req;
 			strncpy(req.ifr_name, ifname, IFNAMSIZ);
+			req.ifr_name[IFNAMSIZ - 1] = '\0';
```

Comment:
checking ifname length is within IFNAMESIZ should be considered.
because it could be possible that ifname can be over IFNAMESIZ.

Reply:
@pillip8282 
```
struct ifreq {
	char ifr_name[IFNAMSIZ];	/* Network device name (e.g. "eth0") */
	union {
		struct sockaddr ifru_addr;	/* IP Address */
		struct sockaddr ifru_dstaddr;	/* P-to-P Address */
		struct sockaddr ifru_broadaddr;	/* Broadcast address */
		struct sockaddr ifru_netmask;	/* Netmask */
		struct sockaddr ifru_hwaddr;	/* MAC address */
		int ifru_count;			/* Number of devices */
		int ifru_mtu;			/* MTU size */
		uint8_t ifru_flags;		/* Interface flags */
		struct mii_iotcl_notify_s ifru_mii_notify;	/* PHY event notification */
		struct mii_ioctl_data_s ifru_mii_data;	/* MII request data */
	} ifr_ifru;
};
```
'ifr_name' is defined with IFNAMSIZ